README.md

```diff
@@ -1,52 +1,51 @@
-[![Build Status](https://travis-ci.org/Kinvey/js-sdk.svg?branch=refactor%2FMLIBZ-2587_Improve_SDK)](https://travis-ci.org/Kinvey/js-sdk)
+<p align="left">
+  <a href="https://www.progress.com/kinvey" style="display: inline-block;">
+    <img src="logo-progresskinvey.png">
+  </a>
+</p>
 
-# Overview
+<p align="left">
+  <a href="https://cirrus-ci.com/github/Kinvey/js-sdk"><img alt="Build Status" src="https://api.cirrus-ci.com/github/Kinvey/js-sdk.svg"></a>
+  <a href="https://conventionalcommits.org"><img alt="Conventional Commits" src="https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg"></a>
+</p>
 
-[Kinvey](http://www.kinvey.com) (pronounced Kin-vey, like convey) makes it ridiculously easy for developers to setup, use and operate a cloud backend for their mobile apps. They don't have to worry about connecting to various cloud services, setting up servers for their backend, or maintaining and scaling them.
+# Overview
 
-The JS SDK is used as a way to share common code for platform specific SDK shims. Here is a list of the platforms we currently support.
+[Kinvey](https://www.progress.com/kinvey) is a high-productivity serverless application development platform that provides developers tools to build robust, multi-channel applications utilizing a cloud backend and front-end SDKs. As a platform, Kinvey provides many solutions to common development needs, such as a data store, data integration, single sign-on, and file storage. With Kinvey, developers can focus on building what provides value for their app - the user experience (UX) and business logic of the application. This approach increases developer productivity and aims to enable higher quality apps by leveraging Kinvey's pre-built components.
 
-* [Angular](packages/angular-sdk)
-* [HTML5](packages/html5-sdk)
-* [NativeScript](packages/nativescript-sdk)
-* [NativeScript Angular](packages/nativescript-angular-sdk)
-* [NodeJS](packages/node-sdk)
+## Platforms Supported
 
-## Thanks
-Thanks to [BrowserStack](http://browserstack.com) for providing us with a free account.
-<p align="left">
-  <a href="http://browserstack.com" style="display: inline-block;">
-    <img src="logo-browserstack.png">
-  </a>
-</p>
+We currently support the following JavaScript platforms:
 
-## License
-See [LICENSE](LICENSE) for details.
+- [Angular](packages/angular-sdk)
+- [NodeJS](packages/node-sdk)
+- [React](packages/html5-sdk)
```

Comment:
Do we need to mention other web-based platforms here, for use with HTML5 SDK?

Reply:
I added `Vue` and `Web JavaScript`. Do we want to call it `HTML5` instead of `Web JavaScript`?

Reply:
Yes let's keep it `HTML5` for now. Changing it would require other changes to DevCenter.

Reply:
Ok I fixed it in ba0e37e.